src/main/scala/org/allenai/plugins/DockerBuildPlugin.scala

```diff
@@ -765,7 +765,9 @@ $DOCKERFILE_SIGIL
         case (hostPort, containerPort) => Seq("-p", s"$hostPort:$containerPort")
       }
       // Start up the container.
-      Process(baseCommand ++ portArgs ++ dockerRunFlags.value :+ mainImageName.value).run()
+      val fullCommand = baseCommand ++ portArgs ++ dockerRunFlags.value :+ mainImageName.value
+      logger.info("Running command: " + fullCommand.mkString(" "))
```

Comment:
I notice that there's a log message on `753` that also gets printed. Maybe also remove that line?